content/community/news/science-updates/2022-12-19-nolasco-blog-post.md

```diff
@@ -0,0 +1,22 @@
+---
+date: "2022-12-19T00:00:00+00:00"
+title: 'Open Science'
+tags: ["force field", "open science", "Open Force Field", "OpenFF" ]
+categories: ["science"]
+draft: false
+description: 'By working together, we can make this happen'
+slug: 'open-science-2022-12-19'
+weight: 10
+markup: markdown
+thumb: "OpenFF-Initiative.png"
+author: "Diego Nolasco"
+---
+
+Science plays a crucial role in shaping our future and increasing knowledge is key to this progress. This is why initiatives like the Open Force Field, which aim to turn cutting-edge scientific research into valuable technology, need ongoing support. By providing long-term support to these initiatives, we can ensure that they can continue to develop and deliver comprehensive solutions that are accessible to all. It is important to recognize that the success of initiatives like the Open Force Field relies on ongoing interest and support.
+
+The concept of quality can vary depending on the context in which it is applied. In science, quality is often linked to our ability to continually add value as our understanding and capabilities evolve. In order to maintain and improve the quality of force fields, sustained investment is necessary. This is especially important if we want these force fields to keep up with the advancements in science and meet our changing needs. It is important to continue investing in initiatives like the Open Force Field, as the development of new science takes time and today's efforts can lead to progress in the future. By investing in the Open Force Field, we can ensure that we can continue to benefit from the work that has been started.
+
+It is often thought that something is perfect and cannot be improved upon when it is believed to provide sufficient benefits. However, this mindset can be limiting and prevent us from pushing the boundaries of what is possible. By considering current technology and science as sufficient, we may stop putting effort into advancing and maintaining it, leading to a loss of expertise and the deterioration of infrastructure. This can ultimately result in costly problems in the future when we are faced with challenges that require new solutions. Instead of settling for what is currently available and choosing short-term cost-saving measures, it is important to continue striving for progress and innovation.
```

Comment:
Could be more explicit about how this relates to OpenFF, e..g after the first sentence, "For example, some might argue that OpenFF has already achieved enough because its force fields have substantially improved from its starting point." Then at the end of the paragraph you could say something like, "In OpenFF in particular, we see a wide variety of opportunities to further improve the accuracy and transferability of our force fields, and thus continued community support is needed so that we can deliver these benefits." Not required, but right now one might miss how this paragraph is supposed to connect to OpenFF.